src/tsickle.ts

```diff
@@ -888,6 +888,36 @@ class ExternsWriter extends ClosureRewriter {
       this.writeExternsFunction(name.getText(), paramNames, namespace);
     }
 
+    // Handle method declarations/signatures first, since we need to deal with overloads.
+    // TODO: Build a collection of the remaining members
+    let methods: {[id: string]: ts.MethodDeclaration[];} = {};
```

Comment:
Also, habit. When I've used this ES5 dictionary/map pattern, I've typically left it as id or key, since that is what it is acting as. Changed for clarity here.